pom.xml

```diff
@@ -164,6 +238,33 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>com.github.eirslett</groupId>
+          <artifactId>frontend-maven-plugin</artifactId>
+          <version>${version.frontend-maven-plugin}</version>
+          <configuration>
+            <nodeVersion>${version.node}</nodeVersion>
+            <npmVersion>${version.npm}</npmVersion>
+            <yarnVersion>${version.yarn}</yarnVersion>
+          </configuration>
+        </plugin>
+        <plugin>
+          <groupId>org.asciidoctor</groupId>
+          <artifactId>asciidoctor-maven-plugin</artifactId>
+          <version>2.0.0-RC.1</version>
```

Comment:
I think that the prod rule is "no fixed versions of anything".

Reply:
Copied from kie-parent (needed to set a lot of dependency versions to make it build again)

Reply:
Right. My point is this should also be specified as a property.
Alternatively we could use the `optaplanner-build-parent`, but I'm not sure if that is proper. We'd have to ask Geoffrey what his idea for that is.